core/src/main/resources/error/error-classes.json

```diff
@@ -482,6 +482,12 @@
     ],
     "sqlState" : "22012"
   },
+  "DUPLICATED_MAP_KEY" : {
```

Comment:
Is this associated with a function? Can we name it?

Reply:
I believe it seems not related to specific function, but not 100% sure since I haven't enough context for Scala-side functions yet.
@MaxGekk Could you have an answer for this, if we name any specific function for this error class?

Reply:
This error can occur in multiple functions:
- map_concat
- map_from_entries
- map
- map_from_arrays
- str_to_map
- transform_keys

Reply:
Yeah, IMHO maybe it is okay to leave it as is, because I think "DUPLICATED_MAP_KEY" is pretty intuitive when users face this error during use such a map function?